Untitled.ipynb

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Remove this file before merging

Reply:
Thank you, I had not set my `.gitignore_global` file in the new computer yet